homework1/array/array.c

```diff
@@ -0,0 +1,60 @@
+﻿#include <stdio.h>
+#include <stdbool.h>
+
+#define ARRAYSIZE 11
+
+// Тело отражающей функции.
+// 
+// Принимает на вход указатель на массив и номера первого и последнего
+// элементов участка массива, который нужно отразить.
+//
+// Меняет местами первый и последний элемент участка, затем второй и предпоследний и т. д.
+void reverse(int* arrayForReverse, int firstReverseElement, int lastReverseElement) {
+    char temporary = 0;
+    int operationCounter = 0;
+
+    for (int i = firstReverseElement - 1; i < (firstReverseElement + ((lastReverseElement - firstReverseElement) / 2)); ++i) {
+        temporary = arrayForReverse[i];
+        arrayForReverse[i] = arrayForReverse[lastReverseElement - 1 - operationCounter];
+        arrayForReverse[lastReverseElement - 1 - operationCounter] = temporary;
+
+        operationCounter++;
+    }
+}
+
+// Распечатывает массив фиксированного размера, на вход принимает указатель на него.
+void printArray(int* arrayToPrint) {
+    for (int i = 0; i < ARRAYSIZE; ++i)
+        printf("%d ", arrayToPrint[i]);
```

Comment:
```suggestion
    for (int i = 0; i < ARRAYSIZE; ++i) {
        printf("%d ", arrayToPrint[i]);
    }
```
По нашему стайлгайду фигурные скобки ставятся всегда вокруг тел операторов типа if, for, while и т.п., даже если внутри всего один оператор. Тут и везде надо поправить.